packages/service-client/odsp-client/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: [require.resolve("@fluidframework/eslint-config-fluid/minimal"), "prettier"],
```

Comment:
We should start with the recommended config at least, rather than the minimal one.
```suggestion
	extends: [require.resolve("@fluidframework/eslint-config-fluid"), "prettier"],
```